test/services/PayeAtsServiceSpec.scala

```diff
@@ -49,7 +49,8 @@ class PayeAtsServiceSpec extends UnitSpec with MockitoSugar with GuiceOneAppPerT
 
   val mockMiddleConnector = mock[MiddleConnector]
   implicit val request = PayeAuthenticatedRequest(testNino, FakeRequest("GET", "/annual-tax-summary/paye/"))
-  lazy val mockAuditService: AuditService = mock[AuditService]
+  //ToDo need to find what is causing intermittent failure for this unit test
```

Comment:
Did you find what caused it or is this still a problem?

Reply:
it seems like removing lazy val has fixed it as i have not faced this issue since then.
Will do the investigation if it fails again as it is an intermittent issue